src/index.html

```diff
@@ -1,13 +1,54 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="productcard" data-qa="card">
+      <div class="images">
+        <img
+          class="images_item"
+          src="images/imac.jpeg"
+          alt="foto"
+      >
+      </div>
```

Comment:
it is a redundant wrapper plus name your classes using BEM methodology
```suggestion
        <img
          class="card__image"
          src="images/imac.jpeg"
          alt="foto"
        >
```